tests/autoyast/verify_system_lvm.pm

```diff
@@ -0,0 +1,45 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+#
+# Summary: Basic verification of system after autoyast installation
+# with lvm on btrfs.
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use strict;
+use warnings;
+use parent 'installbasetest';
+use testapi;
+
+sub run {
+
+    # Check if AutoYaST displayed a warning about shrinking partitions to make them
+    # fit (bsc#1078418).
+    record_info("Warning", "Warning displayed for partition shrinking")
+      unless (script_run("zgrep \"Some additional space\" /var/log/YaST2/y2log*.gz"));
```

Comment:
I believe that we should use `assert_script_run`. Recording info won't be visible in case of failures.

Reply:
I am sure if the lack of warnings would definitely mean that something went wrong. Do you know for sure that this is expected or should I ask yast team?

Reply:
Yes, we expect this warning to be shown, so as long as there is no regression, it has to be shown. We also indirectly assert this during installation, but we use generic needle like `known-warning`.

Reply:
ok